torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -260,14 +307,26 @@ class TensorExprFuser {
     }
     auto subgraph = SubgraphUtils::getSubgraph(n);
     size_t num_modes = blockSize(subgraph->block());
-    if (num_modes < minSubgraphSize_) {
+    if (num_modes < min_group_size_) {
       GRAPH_UPDATE("Fusion group is too small, unmerging: ", *n);
       SubgraphUtils::unmergeSubgraph(n);
       return true;
     }
     return false;
   }
 
+  void updateTypeinfoMapWithVmap(
+      const std::unordered_map<Value*, Value*>& vmap) {
```

Comment:
i'd love to see an assertion here to double check that we don't have the same address as both a key and value. Took two hours out of my life. But we could do it later.